server/schedule/metrics.go

```diff
@@ -56,7 +56,7 @@ var (
 			Subsystem: "schedule",
 			Name:      "store_limit",
 			Help:      "Limit of store.",
-		}, []string{"store", "type"})
+		}, []string{"store", "type", "limit_type"})
```

Comment:
Would you like to help us improve these metrics? The metrics currently are not very useful.

Reply:
I think I can solve this issue in another PR.